UndertaleModCli/Program.cs

```diff
@@ -657,7 +657,8 @@ private void DumpAllTextures()
         {
             if (Verbose)
                 Console.WriteLine($"Dumping {texture.Name}");
-            File.WriteAllBytes($"{directory}/{texture.Name.Content}.png", texture.TextureData.TextureBlob);
+            using FileStream fs = new($"{directory}/{texture.Name.Content}.png", FileMode.Create);
+            fs.Write(texture.TextureData.Image.ConvertToPng().ToSpan());
```

Comment:
Can't you just:
```suggestion
            using FileStream fs = new($"{directory}/{texture.Name.Content}.png", FileMode.Create);
            texture.TextureData.Image.SavePng(fs));
```

Reply:
Good catch, I've now changed both occurrences of this.